satellite/overlay/config.go

```diff
@@ -20,7 +20,8 @@ var (
 // Overlay cache responsibility.
 type Config struct {
 	Node                 NodeSelectionConfig
-	UpdateStatsBatchSize int `help:"number of update requests to process per transaction" default:"100"`
+	UpdateStatsBatchSize int           `help:"number of update requests to process per transaction" default:"100"`
+	UptimeCacheDuration  time.Duration `help:"the length of time to cache uptime before updating the overlay cache again" releaseDefault:"1m" devDefault:"1s"`
```

Comment:
when this is set to 0 does it just constantly update or never update?

Reply:
Constantly update.

Reply:
Wondering whether there's a better name... maybe `CoalesceDuration`, `UptimeFlushInterval`, `UptimeFlushDelay`?